prov/sockets/src/sock_ep_msg.c

```diff
@@ -243,29 +243,31 @@ static int sock_ep_cm_getname(fid_t fid, void *addr, size_t *addrlen)
 {
 	struct sock_ep *sock_ep = NULL;
 	struct sock_pep *sock_pep = NULL;
+	size_t len;
 
 	if (*addrlen == 0) {
 		*addrlen = sizeof(struct sockaddr_in);
 		return -FI_ETOOSMALL;
 	}
 
-	*addrlen = MIN(*addrlen, sizeof(struct sockaddr_in));
-
+	len = MIN(*addrlen, sizeof(struct sockaddr_in));
```

Comment:
This function looks correct to me.  As an alternative, you could do this to have a single exit point:

len = MIN(...)
if (len ==)
   goto out;   /* jumps to setting addrlen below */